scripts/setup-local-test-cluster.sh

```diff
@@ -62,6 +63,9 @@ helm upgrade --install loki-otlp grafana/grafana-agent -f "${GRAFANA_AGENT_LOKI_
 echo "Deploying Tempo..."
 helm upgrade --install tempo grafana/tempo -n tempo --create-namespace --wait
 
+echo "Deploying Pyroscope..."
+helm upgrade --install pyroscope grafana/pyroscope -f "${PYROSCOPE_VALUES}" -n pyroscope --create-namespace --wait
```

Comment:
By default this will not support multiple tenants. You will need to supply a extraArg like that:

```
--set pyroscope.extraArgs."auth.multitenancy-enabled"=true
```